ui/app/styles/components/confirm.scss

```diff
@@ -37,16 +37,16 @@
   button.link,
   a {
     background-color: $white;
-    color: $text;
+    color: $grey-darkest;
 
     &:hover {
```

Comment:
removed all `menu-item-xx` vars as they were not used consistently and did not provide clarity but rather confusion on what it was. If we had always used this var to declare the color of a menu-item that would be another story, but we don't and we didn't.